benchmark/Project.toml

```diff
@@ -1,13 +1,20 @@
 [deps]
 AMD = "14f7f29c-3bd6-536c-9a0b-7339e30b5a3e"
+ArgParse = "c7e460c6-2fb9-53a9-8c5b-16f535851c63"
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
Still needed?

Reply:
Absolutely. Argparse and BenchmarkTools are required to run *send_comment_to_pr.jl* and *run_benchmarks.jl* respectively.